### DISCUSSION TITLE
“Universal Silicon Labs Flasher” for Silabs OTW firmware update

### DISCUSSION THREAD

**Discussion by Hedda** [grobasoz/zigbee-firmware#37] [General]
FYI, noticed that puddly is developing a "Universal Silicon Labs Flasher" for Silabs OTW firmware update:

https://github.com/puddly/universal-silabs-flasher

Looks like it will be used in a future Home Assistant Add-on for easily flashing [SkyConnect](https://www.home-assistant.io/skyconnect/) and [Yellow](https://www.home-assistant.io/blog/2021/09/13/home-assistant-yellow/):

https://github.com/home-assistant/addons-development/pull/86

PS: See related addons:

https://github.com/home-assistant/addons/tree/master/silabs-multiprotocol

https://github.com/home-assistant/addons-development/tree/master/silabs-concurrent-multiprotocol

**Comment by MattWestb** [comment]
I think his **black** side you can see here https://github.com/puddly/deconz-network-key-leak-poc.
I like our Puddly :-))))))))
